.github/workflows/pages.yml

```diff
@@ -26,4 +26,10 @@ jobs:
         uses: peaceiris/actions-gh-pages@v3
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
-          publish_dir: ./docs
\ No newline at end of file
+          publish_dir: ./docs
+
+      - name: Purge TQ backend assets
+        uses: fjogeleit/http-request-action@v1
```

Comment:
We should pin to a specific commit instead of a version tag like we did in `twilioquest-base` here:
https://github.com/TwilioQuest/twilioquest-base/blob/main/.github/workflows/publish-version-base.yml#L45

Apparently, tagged reference code can be changed without our knowledge and potentially introduce a vulnerability.

Reply:
Make sense, I would rather use a CLI command then. Thanks for noticing this 💯